bin/assign_entity_weights.py

```diff
@@ -25,34 +25,33 @@
 
 def parse_args(args=None):
     parser = argparse.ArgumentParser()
-    parser.add_argument('-m', "--microbiome-ids", metavar="ID", nargs="*", type=int, help="List of microbiome IDs")
-    parser.add_argument('-w', "--weights-files", metavar="PATH", nargs="*", type=str, help="List of corresponding weights files (entity_name, weight)")
-    parser.add_argument('-o', "--out", required=True, type=argparse.FileType('w'), help="Output TSV file (entity_name, microbiome_id, entity_weight)")
+    parser.add_argument('-w',   "--weights",            required=True, metavar="PATH",  type=str,                       help="Path to the weights input file")
+    parser.add_argument('-cw',  "--conditions_weights", required=True, metavar="PATH",  type=str,                       help="Path to the condition weights map input file")
+    parser.add_argument('-o',   "--out",                required=True,                  type=argparse.FileType('w'),    help="Output TSV file (condition_id, entity_name, entity_weight)")
     return parser.parse_args(args)
 
 args = parse_args()
 
-# Check user arguments for validity
-if len(args.microbiome_ids) != len(args.weights_files):
-    sys.exit("The number of microbiome IDs and weights files has to be identical!")
+weights_input = pd.read_csv(args.weights, sep="\t")
```

Comment:
I tried to keep the naming/description consistent with what was there before, e.g. in prepare_entity binding_ratios.py:
```
parser.add_argument("-c"     , "--conditions"               , help="Path to the conditions input file"                      , type=str   , required=True)
parser.add_argument("-cam"   , "--condition-allele-map"     , help="Path to the condition allele map input file"            , type=str   , required=True)
```
Since --weights is the same kind of db_table as conditions, I named and described it accordingly. But I can change weights_input to just weights so it's named the same as the argument and I'll list the columns in the description